dataset_loaders/data_augmentation.py

```diff
@@ -415,6 +426,18 @@ def random_transform(x, y=None,
         end of the channel axis of the image. If True, angle and
         magnitude will be returned, if set to 'rbg' an RGB representation
         will be returned instead. Default: False.
+    repeat_first_optical_flow: bool
+        If True the optical flow for the first frame of the whole video
+        sequence will be duplicated, otherwise it will be an array of zeros
+        with the same shape of the first frame.
+    compute_optical_flow: bool
+        If not False the optical flow is computed online for each
+        sequence to be loaded. If True the optical flow is loaded from
+        the specific directory (if exists).
+    optical_flow_type: string
+        Indicates the method used to generate the optical flow. The
+        optical flow is loaded from a specific directory based on this
+        type.
```

Comment:
Not sure what all of these mean, but I'll comment on the docstrings once you fix the code.